cmd/preview/deploy.go

```diff
@@ -49,6 +49,7 @@ type DeployOptions struct {
 	timeout            time.Duration
 	variables          []string
 	wait               bool
+	members            *[]string
```

Comment:
Why are we setting a pointer here? If we set the pointer here we should be checking if the pointer is nil across the code.

Checking the `AddMembers` signature it expects a `[]string` type